api/BUILD

```diff
@@ -1,5 +1,17 @@
 load("//bazel:api_build_system.bzl", "api_proto_library")
 
+cc_library(
```

Comment:
Do we want to bind all of these together as one cc_library, or export them individually. I ask because looking at the related envoy change https://github.com/lyft/envoy/pull/1039, when you are depending in eds on api_cc, you bring unrelated proto libraries in. In thinking that bazel is meant to make dependencies explicit, I would think that exporting these individually might make more sense. Thoughts?